cadence/errors.py

```diff
@@ -19,14 +19,14 @@ class InternalServiceError(Exception):
 
 @dataclass
 class DomainAlreadyExistsError(Exception):
-    message: str
+    message: str = ""
```

Comment:
just curios, is there any difference or just a more standard way to do it?

Reply:
The empty string is the default value, if I try to create a new instance of `DomainAlreadyExistsError` without default value it will complain.  I think this might be the most standard way to do it, but my lack of python knowledge doesn't let me add more valuable input.